File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/SdkVersionChecker.java
package com.mapbox.services.android.navigation.v5.navigation;

class SdkVersionChecker {

  private final int currentSdkVersion;

  SdkVersionChecker(int currentSdkVersion) {
    this.currentSdkVersion = currentSdkVersion;
  }

  boolean isGreaterThan(int sdkCode) {
    return currentSdkVersion > sdkCode;
  }
}

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/legacy/SdkVersionChecker.kt
package com.mapbox.navigation.ui.legacy

class SdkVersionChecker(private val currentSdkVersion: Int) {

    fun isGreaterThan(sdkCode: Int): Boolean =
        currentSdkVersion > sdkCode

    fun isEqualOrGreaterThan(sdkCode: Int): Boolean =
        currentSdkVersion >= sdkCode
}
